### PR TITLE
Update to 3.0.0-alpha-2

### PR DESCRIPTION
**Pull request by erick0z**
Update runtime, proxy and shim to 3.0.0-alpha.2
- Include a patch for integrating containers/virtcontainers
pause binary in the cc-runtime package.
- Delete virtcontainers-pause package source files.
- Allow custom APIURLs for local OBS instances.

Fixes https://github.com/clearcontainers/packaging/issues/9
Fixes https://github.com/clearcontainers/packaging/issues/4

### CODE REVIEW

**Comment by gorozco1** [comment]
@erick0z Please remove virtcontainers-pause and add https://github.com/clearcontainers/runtime/pull/276 as a patch to this PR.

**Review by gorozco1 — CHANGES_REQUESTED** [review]
Could you also add 
Fixes # in the commit message?

File: versions.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+cc_runtime_hash=f24c2b04f2f8bb76d879564a0a9398ba7d99c4a2
+cc_shim_hash=9d1a3ee050372eb5d4d61fc6011eb698c71a84b2
 virtcontainers_hash=93aa8226e6fa7e9bb2d553b571209f4f94e96348
-cc_proxy_version=3.0.0-alpha.1
-cc_runtime_version=3.0.0-alpha.1
-cc_shim_version=3.0.0-alpha.1
+cc_proxy_version=3.0.0-alpha.2
+cc_runtime_version=3.0.0-alpha.2
+cc_shim_version=3.0.0-alpha.2
 virtcontainers_version=0.7.0
```
Comment (gorozco1) [inline versions.txt]: remove virtcontainers_version

File: versions.txt [hunk 2]
```diff
@@ -1,8 +1,8 @@
-cc_proxy_hash=41b78995e39e3c27478e03f7f34cf01c75f1f7db
-cc_runtime_hash=734bdd9f26967ef86c863b64c71fe38d09c2f39a
-cc_shim_hash=c070a40dcb454634bc88e95cd234d49121aea44d
+cc_proxy_hash=7cb0341712a90d7286fa1730162907456fffda0f
+cc_runtime_hash=f24c2b04f2f8bb76d879564a0a9398ba7d99c4a2
+cc_shim_hash=9d1a3ee050372eb5d4d61fc6011eb698c71a84b2
 virtcontainers_hash=93aa8226e6fa7e9bb2d553b571209f4f94e96348
```
Comment (gorozco1) [inline versions.txt]: remove virtcontainers_hash

**Review by gorozco1 — CHANGES_REQUESTED** [review]
please remove virtcontainers related stuff from versions.txt

File: runtime/cc-runtime-bin.install [hunk 3]
```diff
@@ -0,0 +1,2 @@
+cc-runtime /usr/bin
+pause/pause /var/lib/clear-containers/runtime/bundles/pause_bundle/bin/
```
Comment (gorozco1) [inline runtime/cc-runtime-bin.install]: emacs detected... no end of file

File: runtime/cc-runtime-config.install [hunk 4]
```diff
@@ -0,0 +1 @@
+config/configuration.toml /etc/clear-containers/configuration.toml
```
Comment (gorozco1) [inline runtime/cc-runtime-config.install]: emacs detected... no end of file

File: runtime/cc-runtime.dsc-template [hunk 5]
```diff
@@ -12,6 +12,20 @@ DEBTRANSFORM-RELEASE: 1
...
 Description:
  An Open Containers Initiative (OCI) "runtime" that launches an Intel VT-x secured Clear Containers 3.0 hypervisor, rather than a standard Linux container.
+ This package contain the cc-runtime and the pause binaries.
+
+Package: cc-runtime-config
+Architecture: amd64
+Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}
+Description:
+ An Open Containers Initiative (OCI) "runtime" that launches an Intel VT-x secured Clear Containers 3.0 hypervisor, rather than a standard Linux container.
+ This package contain the configuration file.
```
Comment (gorozco1) [inline runtime/cc-runtime.dsc-template]: emacs detected... no end of file

File: runtime/debian.control-template [hunk 6]
```diff
@@ -10,7 +10,20 @@ DEBTRANSFORM-RELEASE: 1
...
 Description:
  An Open Containers Initiative (OCI) "runtime" that launches an Intel VT-x secured Clear Containers 3.0 hypervisor, rather than a standard Linux container.
+ This package contain the cc-runtime and the pause binaries.
 
+Package: cc-runtime-config
+Architecture: amd64
+Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}
+Description:
+ An Open Containers Initiative (OCI) "runtime" that launches an Intel VT-x secured Clear Containers 3.0 hypervisor, rather than a standard Linux container.
+ This package contain the configuration file.
```
Comment (gorozco1) [inline runtime/debian.control-template]: emacs detected... no end of file

File: runtime/debian.series [hunk 7]
```diff
@@ -0,0 +1 @@
+build-Add-containers-virtcontainers-pause-binary.patch
```
Comment (gorozco1) [inline runtime/debian.series]: emacs detected... no end of file

**Review by gorozco1 — CHANGES_REQUESTED** [review]
end of files

File: runtime/cc-runtime-bin.install (hunk 3 above)
Comment (erick0z) [reply runtime/cc-runtime-bin.install]: oops

**Review by gorozco1 — APPROVED** [review]

**Pull request merged**